src/cms/data/functions/loadOperationData.js

```diff
@@ -0,0 +1,220 @@
+            } else {
+              if (doesExist) {
+                if (allowUpdates) { // update existing record
+                  const u = await modelService.update({ id: existingRecord.id }, data);
+                  updateCount++;
+                }
+              } else { // create a new record
+                const c = await modelService.create(data);
+                createCount++;
+              }
+            }
+          }
+
+        } catch (e) {
+          console.log("!! Json import failed for " + modelName + " data with " + idName + ":" + idVal);
+          console.log(e);
+        }
+      };
+      console.log(modelName + " - Updates: " + updateCount + ", Creates: " + createCount + ", Missing: " + missingCount);
+    }
+    return(true)
+
+  } catch (error) {
+    console.log(`error loading ${modelName}...`);
+    console.log(error);
+    return(false)
+  }
+}
+
+const loadParkOperations = async ({ isSeedMode = true, allowUpdates = false }) => {
+  
+  const a = await updateOperationsFromJson({
+    modelName: "park-operation",
+    jsonFile: "./data/park-operation.json",
+    objectName: "parkOperation",
+    idName: "orcs",
+    isSeedMode: isSeedMode,
+    allowUpdates: allowUpdates,
+  });
+  return(a)
+};
+
+const loadParkOperationSubAreaTypes = async ({ isSeedMode = true, allowUpdates = false }) => {
+  const a = await updateOperationsFromJson({
+    modelName: "park-operation-sub-area-type",
+    jsonFile: "./data/park-operation-sub-area-types.json",
+    objectName: "parkOperationSubAreaTypes",
+    idName: "subAreaTypeId",
+    isSeedMode: isSeedMode,
+    allowUpdates: allowUpdates,
+  });
+  return(a)
+};
+
+const loadParkOperationSubAreas = async ({ isSeedMode = true, allowUpdates = false }) => {
+  const a = await updateOperationsFromJson({
+    modelName: "park-operation-sub-area",
+    jsonFile: "./data/park-operation-sub-areas.json",
+    objectName: "parkOperationSubAreas",
+    idName: "parkSubAreaId",
+    isSeedMode: isSeedMode,
+    allowUpdates: allowUpdates,
+  });
+  return(a)
+};
+
+const loadParkOperationSubAreaDates = async ({ isSeedMode = true, allowUpdates = false }) => {
+    const a = await updateOperationsFromJson({
+    modelName: "park-operation-sub-area-date",
+    jsonFile: "./data/park-operation-sub-area-dates.json",
+    objectName: "parkOperationSubAreaDates",
+    idName: "parkSubAreaId", // exception to pattern - at most one record per subArea per year
+    isSeedMode: isSeedMode,
+    allowUpdates: allowUpdates,
+  });
+}
+
+const loadData = async ({ isSeedMode = true, allowUpdates = false }) => { 
+  const config = { isSeedMode: isSeedMode, allowUpdates: allowUpdates }
+  try {
+    const p1 = await loadParkOperationSubAreaTypes(config)
+    const p2 = await loadParkOperationSubAreas(config)
+    const p3 = await loadParkOperationSubAreaDates(config)
+    const p4 = await loadParkOperations(config)
+  } catch (err) {
+    console.log("Error in loading operations data")
```

Comment:
We should output error here instead of being silent.

Reply:
updated